ansible/roles/common/tasks/main.yml

```diff
@@ -60,3 +60,22 @@
     line: "session required pam_limits.so"
   tags:
     - nofile_limits
+
+
+- name: "Sets hostname"
+  become: yes
+  hostname:
+    name: "{{ hostname }}"
```

Comment:
Where is this defined? Some of already set hostnames on these servers might not match with what you have set in the DNS. May be double check this before deploying?